apps/prairielearn/src/pages/instructorQuestions/instructorQuestions.ts

```diff
@@ -0,0 +1,84 @@
+import ERR from 'async-stacktrace';
+import { Router } from 'express';
+import error = require('@prairielearn/error');
+import { logger } from '@prairielearn/logger';
+import sqldb = require('@prairielearn/postgres');
+import { QuestionAddEditor } from '../../lib/editors';
+import fs = require('fs-extra');
+import async = require('async');
+import { QuestionsPage } from './instructorQuestions.html';
+import { getQuestions } from '../../models/questions';
+
+const router = Router();
+const sql = sqldb.loadSqlEquiv(__filename);
+
+router.get('/', function (req, res, next) {
+  let questions;
+  async.series(
+    [
+      (callback) => {
+        fs.access(res.locals.course.path, (err) => {
+          if (err) {
+            if (err.code === 'ENOENT') {
+              res.locals.needToSync = true;
```

Comment:
Where possible, we've been refactoring things to avoid needing to put data in `res.locals`. In some cases that's currently a huge lift, but in this case, it should be pretty straightforward to pass `needToSync` directly to the TypeScript HTML template function.

Reply:
I did refactor a lot of these during this PR but I guess I missed this one. 

Fixed now, thanks!